.travis.yml

```diff
@@ -18,8 +18,8 @@ addons:
     - mysql-client-core-5.6
     - mysql-client-5.6
 rvm:
-  - 2.4.0-preview2
-  - 2.3.1
+  - 2.4.0-preview3
```

Comment:
Actually `2.4.0-rc1` is available starting today! I wonder if `2.4` works as an alias yet. I know `2.3` works as an alias for a floating most-recent ruby-2.3.x.